### ISSUE TITLE
16Bits Number Grouping 

### ISSUE THREAD

**Issue by Domsed** [MaximumOctopus/LEDMatrixStudio#1]
Please, can you add 16Bits Number grouping option to Export function? The ESP32-HUB75-MatrixPanel-DMA library support only 16bits and i cant get right code for it :( . Thanks D.S 

![Screenshot_5](https://user-images.githubusercontent.com/18630640/220329649-aa52ec34-7093-4baa-96c8-721533de1b7b.png)

**Comment by MaximumOctopus** [comment]
I can add that.

RGB colours are 24 bits, so how would a 16 bit output work?

Two groups of 16 bits per colour (based on the colour format selection), eg: 0R GB
Where 0 is a zero byte.

**Comment by MangMuang** [comment]
> I can add that.
> 
> RGB colours are 24 bits, so how would a 16 bit output work?
> 
> Two groups of 16 bits per colour (based on the colour format selection), eg: 0R GB Where 0 is a zero byte.

I think it's called RGB565 which is 16Bit RGB, ImageConverter565 by Henning Karlsen doing that. I think it's good to have RGB565 since it save a lot of programming space. Not sure how to encode it but I do this to decode into RGB888.

unsigned char redA   = ((((ColorIn>> 11) & 0x1F) * 527) + 23) >> 6;
unsigned char greenA = ((((ColorIn>> 5) & 0x3F) * 259) + 33) >> 6;
unsigned char blueA  = (((ColorIn& 0x1F) * 527) + 23) >> 6;

**Comment by MaximumOctopus** [comment]
Yeah I can add that, good idea! I'll get to work on it this week.
